plugin/src/main/scala/chisel3/internal/plugin/ChiselPlugin.scala

```diff
@@ -8,12 +8,16 @@ import nsc.plugins.{Plugin, PluginComponent}
 import scala.reflect.internal.util.NoPosition
 import scala.collection.mutable
 
-private[plugin] case class ChiselPluginArguments(
+private[plugin] class ChiselPluginArguments(
   val skipFiles: mutable.HashSet[String] = mutable.HashSet.empty) {
+  var deprecateSFC: Boolean = true
   def useBundlePluginOpt = "useBundlePlugin"
   def useBundlePluginFullOpt = s"-P:${ChiselPlugin.name}:$useBundlePluginOpt"
   def genBundleElementsOpt = "genBundleElements"
   def genBundleElementsFullOpt = s"-P:${ChiselPlugin.name}:$genBundleElementsOpt"
+  // Use to not emit deprecation warning for `import firrtl._`
+  def omitSFCDeprecation = s"omitSFCDeprecation"
```

Comment:
```suggestion
  def omitSFCDeprecation = s"supressSFCDeprecation"
```
`omit` is a bit too close to `emit` and just saying it out loud sort of confused me 😂